### PR TITLE
Add silicon human spremacy override config, fixes #68957

### LINKED ISSUE #68957: "Asimov Superiority" species perk for humans doesn't show if custom laws are set, which is a problem for production where the custom lawset is Asimov++
https://github.com/tgstation/tgstation/blob/4ad01aa4ce1190af63a1f0914062cb7c190d9d14/code/modules/mob/living/carbon/human/species_types/humans.dm#L72-L79

This was smart foresight (I think I put it in a comment before we moved it to DM) but it means that on live, where we have custom AI laws set to Asimov++, that this perk doesn't show up, despite humans very much being in control.

In addition to this check, there should be another fallback config option that enables this perk, for server operators to be able to say "it's not Asimov, but humans are still superior"

### PR DESCRIPTION
**Pull request by RandomGamer123**
## About The Pull Request
See title

## Why It's Good For The Game
Fixes #68957 

## Changelog

:cl:
fix: Asimov supremacy shows up as a human perk for default servers
config: Added a config option to make Asimov supremacy show up as a perk for humans even if the default lawset is not Asimov (intended for use with Asimov variants, notably Asimov++)
/:cl:

### CODE REVIEW

File: config/game_options.txt [hunk 1]
```diff
@@ -228,6 +228,11 @@ NEAR_DEATH_EXPERIENCE
 
 DEFAULT_LAWS 0
 
+## SILICON HUMAN SUPREMACY OVERRIDE ###
+## This makes "Asimov supremacy" show up as a perk for humans in the character creation menu even if asimov is not the default lawset, such as when used with asimov++
+## Comment this out to make "Asimov Supremacy" show up as a perk for humans
+SILICON_HUMAN_SUPREMACY_OVERRIDE
```
Comment (Mothblocks) [inline config/game_options.txt:234]:
```suggestion
## Uncomment this out to make "Asimov Supremacy" show up as a perk for humans
# SILICON_HUMAN_SUPREMACY_OVERRIDE
```

Comment (Mothblocks) [inline config/game_options.txt:234]: All of this should be "Asimov Superiority", which is what it's actually called.

**Review by Mothblocks — CHANGES_REQUESTED** [review]

File: code/controllers/configuration/entries/game_options.dm [hunk 2]
```diff
@@ -289,6 +289,8 @@
 	min_val = 0
 	max_val = 4
 
+/datum/config_entry/flag/silicon_human_supremacy_override
```
Comment (Mothblocks) [inline code/controllers/configuration/entries/game_options.dm:292]: Document it

File: code/controllers/configuration/entries/game_options.dm [hunk 3]
```diff
@@ -289,6 +289,8 @@
 	min_val = 0
 	max_val = 4
 
+/datum/config_entry/flag/silicon_asimov_superiority_override // Controls if Asimov Superiority appears as a perk for humans even if standard Asimov isn't the default AI lawset
```
Comment (dragomagol) [inline code/controllers/configuration/entries/game_options.dm:292]: The doc should be above the var, and begin with `///`
https://github.com/tgstation/tgstation/blob/master/.github/guides/AUTODOC.md

**Pull request merged**